pmd-core/src/main/java/net/sourceforge/pmd/cpd/CPDTask.java

```diff
@@ -133,6 +133,8 @@ private void report(CPD cpd) throws ReportException {
         CPDRenderer renderer = createRenderer();
         
         try {
+            @SuppressWarnings("PMD.CloseResource")
```

Comment:
This is maybe also a FP. We use the outputstream `os` in a chained BufferedWriter/OutputStreamWriter inside a try-with-resources...